shardingsphere-sql-parser/shardingsphere-sql-parser-statement/src/main/java/org/apache/shardingsphere/sql/parser/sql/common/util/SQLUtil.java

```diff
@@ -73,6 +76,15 @@
     
     private static final String COMMENT_SUFFIX = "*/";
     
+    private static final Map<String, Pattern> PATTERNS = new LinkedHashMap<>();
+    
+    static {
+        PATTERNS.put("$1.", Pattern.compile("^_|([^\\\\])_"));
+        PATTERNS.put("$1.*", Pattern.compile("^%|([^\\\\])%"));
+        PATTERNS.put("_", Pattern.compile("\\\\_"));
+        PATTERNS.put("%", Pattern.compile("\\\\%"));
```

Comment:
Is it better to use variable name instead of map.
Variable name can self describe.